cfgov/v1/wagtail_hooks.py

```diff
@@ -87,30 +86,29 @@ def log_page_deletion(request, page):
 def editor_js():
     js_files = ['js/table-block.js']
 
-    if wagtail.VERSION >= (2, 0):
-        # Temporarily adding Hallo-related JavaScript files to all admin pages
-        # to support the continued use of Hallo in our RichTextTableInput
-        # until we can take more time to migrate that to Draftail.
-        js_files.insert(0, 'wagtailadmin/js/vendor/hallo.js')
-        js_files.insert(0, 'wagtailadmin/js/hallo-plugins/hallo-hr.js')
-        js_files.insert(
-            0,
-            'wagtailadmin/js/hallo-plugins/hallo-requireparagraphs.js'
-        )
-        js_files.insert(
-            0, 'wagtailadmin/js/hallo-plugins/hallo-wagtaillink.js')
-        js_files.insert(
-            0,
-            'wagtaildocs/js/hallo-plugins/hallo-wagtaildoclink.js'
-        )
-        js_files.insert(
-            0,
-            'wagtailembeds/js/hallo-plugins/hallo-wagtailembeds.js'
-        )
-        js_files.insert(
-            0,
-            'wagtailimages/js/hallo-plugins/hallo-wagtailimage.js'
-        )
+    # Temporarily adding Hallo-related JavaScript files to all admin pages
+    # to support the continued use of Hallo in our RichTextTableInput
+    # until we can take more time to migrate that to Draftail.
+    js_files.insert(0, 'wagtailadmin/js/vendor/hallo.js')
+    js_files.insert(0, 'wagtailadmin/js/hallo-plugins/hallo-hr.js')
+    js_files.insert(
+        0,
+        'wagtailadmin/js/hallo-plugins/hallo-requireparagraphs.js'
+    )
+    js_files.insert(
+        0, 'wagtailadmin/js/hallo-plugins/hallo-wagtaillink.js')
```

Comment:
Oops, I messed up when I originally wrote this and didn't put this parenthesis on its own line.

```suggestion
        0, 'wagtailadmin/js/hallo-plugins/hallo-wagtaillink.js'
)
```

Reply:
Done